fmn/rules/taskotron.py

```diff
@@ -5,7 +5,8 @@
 RELEASE_CRITICAL_TASKS = [
```

Comment:
It does make sense to me for this to be in a configuration file, but currently the configuration just using fedmsg's configuration and needs a lot of refactoring anyway. I'll file an issue about it.